packages/property-view/src/browser/property-view-widget-provider.ts

```diff
@@ -20,6 +20,9 @@ import { PropertyDataService } from './property-data-service';
 import { PropertyViewContentWidget } from './property-view-content-widget';
 
 export const PropertyViewWidgetProvider = Symbol('PropertyViewWidgetProvider');
+/**
+ * The `PropertyViewWidgetProvider` should be implemented to provide a new content property view widget for the given selection.
```

Comment:
I'm not sure about the wording here, especially:

> to provide a new content property view widget for the given selection.

```suggestion
 * The `PropertyViewWidgetProvider` should be implemented to provide additional content property view content for the given selection.
```

Reply:
I would maybe go with `provide a property view content widget for the given selection.`.
We use the wording of a property view content widget also elsewhere. An it describes best what this provider does: for the given selection the provider registered, it returns a content widget (that contains the property in an arbitrary form).
What do you think about that?

Reply:
I'm fine with that, I assume the overall idea is that there is a single property-view container, and based on the selection we choose one of many different property-view content widget to be rendered within.